recipe/cross_compile_support.sh

```diff
@@ -71,8 +71,8 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                         (*) echo "" ;;
                     esac)
                 if [[ "${CUDA_MANIFEST_VERSION}" == "" ]]; then
-                    echo 'cross compiling with cuda not in (11.2, 11.8, 12.*) not supported yet'
-                    exit 1
+                    echo "cross compiling with cuda "${CUDA_COMPILER_VERSION}" not supported yet. CUDA 12+ is best supported.";
+                    exit 1;
```

Comment:
Pretty sure we don't need the semicola 🤷

Reply:
Yeah agreed. Michael had them. So I kept them

No strong feelings either way